ihmc-high-level-behaviors/src/libgdx/resources/us/ihmc/rdx/ui/graphics/RDXRawImagePointCloudRenderer.glsl

```diff
@@ -101,7 +101,7 @@ void main()
 #ifdef INPUT_COLOR_IMAGE
    else if (u_coloringMethod == COLOR_FROM_IMAGE)
    {
-      vec3 colorFramePoint = transformPoint3D(depthFramePoint, u_depthToColorTransform);
+      vec3 colorFramePoint = transformPoint3D(depthFramePoint, u_colorToDepthTransform);
```

Comment:
I think you've got it backwards. You're converting the point from depth frame to color frame. The transform that does that should be called depthToColorTransform.

Reply:
Hmm that's true. I'm bad at this stuff so I really don't understand what I'm doing here hahaha. I'll change the names back

Reply:
I can explain on the whiteboard whenever